### PR TITLE
Add options in app settings to check size of media cache and to empty it.

### LINKED ISSUE #6252: Media: Clear Cache
### Details:
Support request from a user: 

```
I have a heavy-image self hosted site and my wordpress app uses sometimes over 1gb of space on my iPhone. To clean this up I have to uninstall it and then install it again... could you make a button to clean the cache of the app without uninstalling it?
```

@SergioEstevao can you please confirm if this is something we're not currently handling?
Thanks!

### PR DESCRIPTION
**Pull request by SergioEstevao**
Fixes #6252 

To test:
 - Open the app
 - Go to a blog with a lot of media
 - Create or edit a post and open the media picker
 - Select the WordPress library
 - Scroll app and down
 - Discard the post
 - Go to Me -> App Settings
 - Check the size displayed on the media cache size row
 - Press the button to Clean Media Cache
 - Check if the size changed
 - Go back to the blog, post and open the WordPress Media library again, and check if everything loads again.

Needs review: @astralbodies

### CODE REVIEW

**Comment by kurzee** [comment]
Note: This will need to be migrated to Swift 3 prior to or once merged into `develop`.

**Comment by SergioEstevao** [comment]
@astralbodies I update/merge to latest develop and upgraded to Swift 3.0 code. so this is ready for another look.

File: WordPress/Classes/Utility/WPImmuTableRows.swift [hunk 1]
```diff
@@ -141,7 +141,7 @@ struct DestructiveButtonRow: ImmuTableRow {
         cell.textLabel?.text = title
         cell.textLabel?.numberOfLines = 0
         cell.textLabel?.lineBreakMode = .byWordWrapping
-
+        cell.selectionStyle = .none
```
Comment (astralbodies) [inline WordPress/Classes/Utility/WPImmuTableRows.swift]: By setting this the row shows nothing when tapping it - was this committed by mistake?

Comment (SergioEstevao) [inline WordPress/Classes/Utility/WPImmuTableRows.swift]: Should we make the button selection or row selection?

File: WordPress/Classes/Extensions/NSFileManager+FolderSize.swift [hunk 2]
```diff
@@ -0,0 +1,86 @@
+import Foundation
+
+
+extension FileManager {
```
Comment (SergioEstevao) [inline WordPress/Classes/Extensions/NSFileManager+FolderSize.swift]: OK

File: WordPress/Classes/Utility/WPImmuTableRows.swift (hunk 1 above)
Comment (astralbodies) [inline WordPress/Classes/Utility/WPImmuTableRows.swift]: Well if we're going to remain consistent with the rest of the settings-like screens making the row tappable makes the most sense.

**Comment by SergioEstevao** [comment]
@astralbodies ready for another look.

**Pull request merged**